MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
```

Comment:
Incomplete rename? Should `SmtpNotificationsController.cs` not be deleted as a counterpart to the addition of this file?